packages/instrumentation-console/package.json

```diff
@@ -18,6 +18,7 @@
   ],
   "scripts": {
     "start": "yarn build -w",
+    "prebuild": "node -p \"'export const VERSION = \\'' + String(require('./package.json').version) + '\\';'\" > src/version.ts",
```

Comment:
Same about moving this into a script.

Will we have different versions for the subpackages vs core package? If not, we can export it once from the core package and then reuse it in subpackages. We can control the exact versions via `peerDependency`

Reply:
yup, will do